lib/Auth/Source/Twitter.php

```diff
@@ -71,8 +60,8 @@ public function __construct(array $info, array $config)
 
         $this->key = $configObject->getString('key');
         $this->secret = $configObject->getString('secret');
+        $this->scope = $configObject->getString('scope');
         $this->force_login = $configObject->getBoolean('force_login', false);
-        $this->include_email = $configObject->getBoolean('include_email', false);
```

Comment:
This is no longer supported by twitter this way.. You have to request them to add your app to a whitelist to be able to retrieve the email address.